### DISCUSSION TITLE
Misurazione quantità di pioggia caduta

### DISCUSSION THREAD

**Discussion by NuccioT** [indomus/forum#2451] [HUB: Home Assistant]
Buongiorno a tutti, sto provando a realizzare un sensore della quantità di pioggia caduta nelle ultime x ore per gestire l’impianto di irrigazione. Allo scopo ho scaricato nel mio HA le seguenti integrazioni: AccuWeather, Met.No, OpenWeather e Weatherbit. I valori di misurazione del sensore rain sono tutti diversi fra loro, nell’ordine: mm/h, mm giornalieri, mm caduti nell’ultima ora, mm di pioggia istantanei. Con la platform statistica o con un paio di template ho cercato di uniformare l’output con nuovi sensori Rain fra le diverse integrazioni ma i valori ottenuti sono molto distanti gli uni dagli altri. Non ho dunque un riferimento affidabile per sapere se interrompere o meno l’irrigazione. Pur ritenendo che tutte le integrazioni forniscano valori affidabili non so a quale fare riferimento. Dal mio punto di vista sarebbe sufficiente sapere quanti mm sono caduti nelle ultime 8 ore per interrompere o meno l’irrigazione per poi riattivarla (o meno) il giorno successivo con una semplice automazione. Avete qualche suggerimento a riguardo? Grazie

**Comment by echopage** [comment]
Se hai zigbee2mqtt puoi provare a crearti il tuo pluviometro! https://community.home-assistant.io/t/diy-zigbee-rain-gauge/255379

**Reply by NuccioT** [reply]
Grazie, ho dato un’occhiata e ci sono spunti interessanti. Grazie mille

**Reply by echopage** [reply]
Penso che il progetto possa essere utilizzato anche con ZHA in fin dei conti si tratta di un sensore porte Aqara modificato!

**Comment by indomus** [comment]
> Buongiorno a tutti, sto provando a realizzare un sensore della quantità di pioggia caduta nelle ultime x ore per gestire l’impianto di irrigazione. Allo scopo ho scaricato nel mio HA le seguenti integrazioni: AccuWeather, Met.No, OpenWeather e Weatherbit. I valori di misurazione del sensore rain sono tutti diversi fra loro, nell’ordine: mm/h, mm giornalieri, mm caduti nell’ultima ora, mm di pioggia istantanei. Con la platform statistica o con un paio di template ho cercato di uniformare l’output con nuovi sensori Rain fra le diverse integrazioni ma i valori ottenuti sono molto distanti gli uni dagli altri. Non ho dunque un riferimento affidabile per sapere se interrompere o meno l’irrigazione. Pur ritenendo che tutte le integrazioni forniscano valori affidabili non so a quale fare riferimento. Dal mio punto di vista sarebbe sufficiente sapere quanti mm sono caduti nelle ultime 8 ore per interrompere o meno l’irrigazione per poi riattivarla (o meno) il giorno successivo con una semplice automazione. Avete qualche suggerimento a riguardo? Grazie


Un suggerimento: https://indomus.it/componenti/netatmo-pluviometro-wireless/
